tests/unit/h_vialib/client_test.py

```diff
@@ -24,15 +24,19 @@ class TestViaClient:
         "via.client.requestConfigFromFrame.origin": "http://random.localhost",
         "via.client.requestConfigFromFrame.ancestorLevel": "2",
         "via.external_link_mode": "new-tab",
+        "via.sec": "signed_token",
     }
     VIA_URL = "http://via.localhost"
     ORIGIN_URL = "http://random.localhost"
 
-    def test_client_default_options(self, client):
+    def test_client_default_options(self, token):
+        token.return_value.create.return_value = "signed_token"
+
+        client = ViaClient(self.VIA_URL, self.ORIGIN_URL, "SECRET")
```

Comment:
I don't think this is any different from the fixture?

```suggestion
    def test_client_default_options(self, client, token):
        token.return_value.create.return_value = "signed_token"
```